src/main/java/com/minecolonies/achievements/MineColoniesAchievementPage.java

```diff
@@ -0,0 +1,27 @@
+package com.minecolonies.achievements;
+
+import net.minecraft.stats.Achievement;
+import net.minecraftforge.common.AchievementPage;
+
+/**
+ * A sublcass for this mod.
+ *
+ * May contain page logic.
+ *
+ * @author Isfirs
+ * @since 0.2
+ */
+public class MineColoniesAchievementPage extends AchievementPage
+{
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)